EdgeMultiplay/CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Changelog
 All notable changes to this EdgeMultiplay Unity Client will be documented in this file.
 
+## [1.2.0] - 2021-05-05
+
+### Fix & Improvements:
+- Fixed App Quiting error on PC/Mac Unity Player.
+- Fixed customizable fallback location for Oculus/Non Phone devices in your GameManager.cs
+```
+EdgeManager.integration.useFallbackLocation = true;
+EdgeManager.integration.setFallbackLocation(longitude, latitude);
+ConnectToEdge(useFallBackLocation:true);
+```
+- Use SendGamePlayEvent() and SendGamePlayEventUDP() in your PlayerManager to SendGamePlayEvents.
+
+### New Features:
+- Server Stats, Now you can see the live server stats from EdgeMultiplay Menu in Unity Editor.
+- New Ownership Transfer options : TakeOver and RequestOwnership.

```

Comment:
New "Orphan Observable"

Reply:
<img width="380" alt="Screen Shot 2021-05-05 at 4 05 45 PM" src="https://user-images.githubusercontent.com/24863504/117220296-ca552780-adbb-11eb-9576-e69abb29d320.png">

Reply:
We had Orphan Observable in the previous release.